app/controllers/two_factor_authentication/webauthn_verification_controller.rb

```diff
@@ -62,7 +62,13 @@ def handle_invalid_webauthn
         )
         redirect_to login_two_factor_webauthn_url(platform: 'true')
       else
-        flash[:error] = t('errors.general')
+        flash[:error] = t(
+          'two_factor_authentication.webauthn_error.connect',
+          link: view_context.link_to(
+            t('two_factor_authentication.webauthn_error.additional_methods_link'),
+            login_two_factor_options_path,
+          ),
+        )
```

Comment:
Does this `%{link}` interpolation get escaped? Or is it fine? I'd expect that the top-level error key would be like `connect_html` and the link would need to be `link_html` for our lints to catch it

I know that our flash components mark content as `.html_safe`  (because strings that are serialized to the `flash` lose their HTML-safeness), so maybe that's why this works? 

https://github.com/18F/identity-idp/blob/main/app/components/flash_component.html.erb#L2

Reply:
I've added consistent _html suffixes.

I was confused for a moment because i18n_spec.rb threw an error during an initial attempt and there are instances in the codebase (found by searching for %{link}) where _html is not added to any of the keys despite some html being added programmatically.